compiler/util.py

```diff
@@ -19,7 +19,11 @@ def pad(lst, index):
 def print_time_delta(prefix, start_time, end_time, args=None):
     ## Always output time in the log.
     time_difference = (end_time - start_time) / timedelta(milliseconds=1)
-    log("{} time:".format(prefix), time_difference, " ms")
+    ## If output_time flag is set, log the time
```

Comment:
Just to clarify, this logs the time regardless of the value of `-d` if we have `--output_time` enabled, right? I think that is the correct behavior!

Reply:
If it is, we probably need to be a bit more clear about it in the comment.

Reply:
That is correct. When using `--output_time`, we get the time for each operation without the need of the `-d` flag!